README.md

```diff
@@ -256,57 +256,86 @@ Check out the [Patient-everything operation spec](https://www.hl7.org/fhir/opera
+
+Example request body for a POST to `http://localhost:3000/4_0_1/$bulk-submit`:
 
 ```json
 {
   "resourceType": "Parameters",
   "parameter": [
     {
-      "name": "input",
-      "part": [
-        {
-          "name": "url",
-          "valueUrl": "http://localhost:3001/ccdc6013-4a14-4bc5-8348-8c4b17a437f7/Coverage.ndjson"
-        },
-        {
-          "name": "inputDetails",
-          "part": [
-            {
-              "name": "resourceType",
-              "valueCode": "Coverage"
-            }
-          ]
-        }
-      ]
+      "name": "manifestUrl",
+      "valueString": "http://localhost:3001/bulkstatus/d1551f79-6afa-40c4-95a1-e92b3394d415"
+    },
+    {
+      "name": "submitter",
+      "valueIdentifier": {
+        "value": "ExampleSubmitterValue"
+      }
     },
     {
-      "name": "input",
-      "part": [
-        {
-          "name": "url",
-          "valueUrl": "http://localhost:3001/ccdc6013-4a14-4bc5-8348-8c4b17a437f7/Condition.ndjson"
-        },
-        {
-          "name": "inputDetails",
-          "part": [
-            {
-              "name": "resourceType",
-              "valueCode": "Condition"
-            }
-          ]
-        }
-      ]
+      "name": "submissionId",
+      "valueString": "Submission2"
+    },
+    {
+      "name": "FHIRBaseUrl",
+      "valueString": "http://localhost:3001"
     }
   ]
 }
 ```
 
-The user can check the status of an $import or $bulk-submit-data request by copying the content-location header in the response, and sending a GET request to `http://localhost:3000/<content-location-header>`.
+#### `$bulk-submit-status`
+
+The server supports the `$bulk-submit-status` endpoint for when a Data Provider wishes to receive updates on the status of a submission after it has kicked off a Bulk Submit operation.
+
+In order to retrieve submission status resources, send a POST request to `http://localhost:3000/4_0_1/$bulk-submit-status` with a FHIR Parameters resource with the following required parameters:
+
+- `submitter` (Identifier)
+- `submissionId` (string)
+
+Example request body for a POST to `http://localhost:3000/4_0_1/$bulk-submit-status`:
+
+```json
+{
+  "resourceType": "Parameters",
+  "parameter": [
+    {
+      "name": "submitter",
+      "valueIdentifier": {
+        "value": "ExampleSubmitterValue"
+      }
+    },
+    {
+      "name": "submissionId",
+      "valueString": "Submission2"
+    }
+  ]
+}
+```
+
+If successful, the operation will respond with a `Content-Location` header with the absolute URL of an endpoint for subsequent status requests (polling location).
+
+#### Bulk Data Status Polling Request
+
+The server follows a FHIR Asynchronous Request Pattern as defined by the [Bulk Submit Draft IG](https://hackmd.io/@argonaut/rJoqHZrPle#Bulk-Data-Status-Polling-Request) for request and response flows following a Bulk Status request.
+
+The server's current implementation returns a partial export manifest and 202
```

Comment:
```suggestion
The server's current implementation returns a partial export manifest and 202 while the submission is incomplete or is being processed. When complete, it returns an export manifest and 200 status.
```

Reply:
Could potentially give more detail here about the export manifest, but it might make sense to wait until we get more information at connectathon prep.